linux_os/guide/services/ssh/ssh_client/ssh_client_rekey_limit/rule.yml

```diff
@@ -54,3 +54,5 @@ fixtext: |-
     Restart the SSH daemon for the settings to take effect.
 
     $ sudo systemctl restart sshd.service
+
+srg_requirement: ''
```

Comment:
add some text

Reply:
I just removed the key since this rule is no longer in the STIG.